opencensus/trace/ext/django/middleware.py

```diff
@@ -218,9 +220,39 @@ def process_request(self, request):
                 SPAN_THREAD_LOCAL_KEY,
                 span)
 
+            if django.VERSION >= (2,):
+                connection.execute_wrappers.append(self._trace_db_call)
```

Comment:
Would this re-append even if the connection already has `self._trace_db_call`?

I tested out locally and it seems like this is the case, I added a simple check to ensure it doesn't do so in a naive way although I'm sure there is a better way to do so.

Reply:
Sounds like a great idea. Thanks @hazzadous